### PR TITLE
Upgrade React + use hooks (!!)

### PR DESCRIPTION
**Pull request by vugman**

### CODE REVIEW

File: src/helpers/dateHelper.js [hunk 1]
```diff
@@ -7,13 +7,11 @@ export function getDay(now = new Date()){
 }
 
 export function incrementDate(date = new Date()){
-    date.setDate(date.getDate()+1);
-    return date;
+    return new Date(date.getTime() + 24 * 60 * 60 * 1000);
```
Comment (itamark) [inline src/helpers/dateHelper.js]: I believe this change would not work in cases of daylight savings time changes (where there's technically 25 hours in a day)

File: src/helpers/dateHelper.js [hunk 2]
```diff
@@ -7,13 +7,11 @@ export function getDay(now = new Date()){
...
 }
 
 export function decrementDate(date = new Date()){
-    date.setDate(date.getDate()-1);
-    return date;
+    return new Date(date.getTime() - 24 * 60 * 60 * 1000);
```
Comment (itamark) [inline src/helpers/dateHelper.js]: Same as above

**Review by itamark — CHANGES_REQUESTED** [review]
Looks good. Just one thing re date.

File: src/helpers/dateHelper.js (hunk 1 above)
Comment (vugman) [reply src/helpers/dateHelper.js]: I'll use the previous impl. 
Only thing is that for the hook to work this methods need to return a new Date object

**Review by itamark — APPROVED** [review]

**Pull request merged**